src/utilities/__tests__/astFromValue-test.js

```diff
@@ -33,10 +33,14 @@ describe('astFromValue', () => {
       { kind: 'BooleanValue', value: false }
     );
 
-    expect(astFromValue(null, GraphQLBoolean)).to.deep.equal(
+    expect(astFromValue(undefined, GraphQLBoolean)).to.deep.equal(
       null
     );
 
+    expect(astFromValue(null, GraphQLBoolean)).to.deep.equal(
```

Comment:
Could you also add a test for `astFromValue(null, new GraphQLNonNull(GraphQLBoolean))` to expect `null`